### PR TITLE
add average physsim speed graph

### PR DESCRIPTION
**Pull request by rajnikantsh**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/2512)

### CODE REVIEW

**Comment by rajnikantsh** [comment]
test!

**Comment by rajnikantsh** [comment]
test!

**Comment by rajnikantsh** [comment]
test!

File: src/main/java/beam/physsim/jdeqsim/AgentSimToPhysSimPlanConverter.java [hunk 1]
```diff
@@ -346,6 +378,11 @@ private Leg createLeg(PathTraversalEvent pte) {
         // end of hack
 
         Route route = RouteUtils.createNetworkRoute(linkIds, agentSimScenario.getNetwork());
+        //Removing first and last link
+        linkIds.removeAll(Lists.newArrayList(route.getStartLinkId(), route.getEndLinkId()));
+        double length = linkIds.stream().mapToDouble(linkId -> networkLinks.get(linkId).getLength()).sum();
+        route.setDistance(length);
```
Comment (REASY) [inline src/main/java/beam/physsim/jdeqsim/AgentSimToPhysSimPlanConverter.java:384]: @wrashid could it possibly affect JDEQSim the presence of distance?

**Pull request merged**